src/utils/supabase/client.test.ts

```
@@ -0,0 +1,1085 @@
+    })
+  })
+
+  describe('Database Operations', () => {
+    describe('Select Operations', () => {
+      it('should perform basic select query', async () => {
+        const mockData = [
+          { id: 1, name: 'Item 1', status: 'active' },
+          { id: 2, name: 'Item 2', status: 'inactive' },
+        ]
+
+        mockSupabaseClient.then = vi.fn().mockResolvedValue({
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)